fitbit/api.py

```diff
@@ -498,7 +493,7 @@ def time_series(self, resource, user_id=None, base_date='today',
         Taking liberty, this assumes a base_date of today, the current user,
         and a 1d period.
 
-        https://wiki.fitbit.com/display/API/API-Get-Time-Series
+        https://dev.fitbit.com/docs/activity/#activity-time-series
```

Comment:
We should include all the time series URLs here, and update "url" in the comment above to be "urls"